wagtailio/project_styleguide/templates/patterns/pages/content_page/content_page.html

```diff
@@ -0,0 +1,22 @@
+{% extends "patterns/base_page.html" %}
+{% load wagtailcore_tags wagtailimages_tags static %}
+
+{% block extra_css %}
+{% endblock %}
```

Comment:
Suggestion: These blocks are meant to override the super (base.html / base_page.html) template. As we're not actively doing anything with them we don't need to add them to the template as it's just additional lines for no functional change.

Reply:
Thanks, done in ae8d891